pom.xml

```diff
@@ -106,6 +106,11 @@
       <version>5.4.1</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+            <groupId>org.rauschig</groupId>
+            <artifactId>jarchivelib</artifactId>
```

Comment:
Indention does not wit with the rest of the file.

Reply:
Done